### PR TITLE
Doc texts updates, update file_id argument

### PR DESCRIPTION
**Pull request by KirillPerepelitsa**
Hi, 

Issue #114,

Reflect all Jeremy text changes, on readme.md

Update on all commands setHelp(), now it will have pattern "Allow you to {command description} api reference {open-api doc link}", instead of "try reboot" or "{open-api doc link}"

### CODE REVIEW

File: src/App/Commands/AppBackups/AppBackupsDeleteCommand.php [hunk 1]
```diff
@@ -24,7 +22,7 @@ protected function configure()
 	{
 		parent::configure();
 		$this->setDescription('Delete an app backup')
-			->setHelp('https://www.lamp.io/api#/app_backups/appBackupsShow')
+			->setHelp('Allow you to delete an app backup, api reference https://www.lamp.io/api#/app_backups/appBackupsShow')
```
Comment (JeremyChase) [inline src/App/Commands/AppBackups/AppBackupsDeleteCommand.php]: @KirillPerepelitsa Please drop `Allow you to` from all setHelps. For example this one should be:

`Delete an app backup, api reference https://www.lamp.io/api#/app_backups/appBackupsShow`

File: src/App/Commands/AppBackups/AppBackupsDescribeCommand.php [hunk 2]
```diff
@@ -25,7 +24,7 @@ protected function configure()
 	{
 		parent::configure();
 		$this->setDescription('Return an app backup')
-			->setHelp('https://www.lamp.io/api#/app_backups/appBackupsShow')
+			->setHelp('Allow you to get an app backup, api reference https://www.lamp.io/api#/app_backups/appBackupsShow')
```
Comment (JeremyChase) [inline src/App/Commands/AppBackups/AppBackupsDescribeCommand.php]: `Get an app backup, api reference https://www.lamp.io/api#/app_backups/appBackupsShow`

File: src/App/Commands/DbBackups/DbBackupsNewCommand.php [hunk 3]
```diff
@@ -20,8 +20,8 @@ class DbBackupsNewCommand extends Command
 	protected function configure()
 	{
 		parent::configure();
-		$this->setDescription('Back up files in database')
-			->setHelp('https://www.lamp.io/api#/db_backups/dbBackupsCreate')
+		$this->setDescription('Back up database')
+			->setHelp('Allow you to back database, api reference https://www.lamp.io/api#/db_backups/dbBackupsCreate')
```
Comment (JeremyChase) [inline src/App/Commands/DbBackups/DbBackupsNewCommand.php]: missing 'up' here. but also, change to:

`backup database, api reference https://www.lamp.io/api#/db_backups/dbBackupsCreate`

**Comment by KirillPerepelitsa** [comment]
@JeremyChase done

File: src/App/Commands/AppRuns/AppRunsListCommand.php [hunk 4]
```diff
@@ -27,7 +27,7 @@ protected function configure()
 	{
 		parent::configure();
 		$this->setDescription('Return all app runs for all user\'s organizations')
-			->setHelp('Allow you to get all app runs for all user\s organizations, api reference https://www.lamp.io/api#/app_runs/appRunsList');
+			->setHelp('Get all app runs for all user\s organizations, api reference' . PHP_EOL . 'https://www.lamp.io/api#/app_runs/appRunsList');
```
Comment (JeremyChase) [inline src/App/Commands/AppRuns/AppRunsListCommand.php]: is this escaping correct?

Comment (KirillPerepelitsa) [reply src/App/Commands/AppRuns/AppRunsListCommand.php]: Escaping character correct, but i've missed quote character, already updated it

**Pull request merged**